embedding/cpp/include/flutter_app.h

```diff
@@ -52,9 +52,6 @@ class FlutterApp : public flutter::PluginRegistry {
  protected:
   void ParseEngineArgs();
 
-  // Whether the app is headed or headless.
-  bool is_headed_ = true;
```

Comment:
Is there any problem with backward-compatibility if the public API is deleted without the deprecation process?

Reply:
I thought there was no problem because I guess the only use of this is Embedding.
However, If the deprecation process is mandatory, I will. please guide me.

Reply:
This public API can be used in application runners exposed to users. If some application is using this API, there will be a problem in build. 
/cc @swift-kim

Reply:
`is_headed_` is an internal API, not a public API. Public APIs are defined in the `public:` section (e.g. `SetDartEntrypoint`).

Also there should be no app (including our internal apps) that directly accesses the `is_headed_` value.

Reply:
The meaning of the `public api` I mentioned is all APIs exposed to the user.
In that sense, protected members also correspond to the public api. But as you said, it's very unlikely that these APIs were used.

Reply:
Although the internal APIs are exposed to users, they are not officially supported and we don't guarantee their forward compatibility.

Reply:
Hey guys... what's the conclusion? Is it correct that the deprecation process is not required in this case?

Reply:
In this case, there seems to be no actual app to use `is_headed_`, so you can ignore the ABI Break. But for the future, it seems necessary to clarify what is internal API or not.
/cc @swift-kim